third_party/proton/dialect/include/Conversion/ProtonGPUToLLVM/GlobalScratchAllocOpToLLVM.h

```diff
@@ -0,0 +1,22 @@
+#ifndef PROTON_GLOBALSCRATCHALLOCOP_TO_LLVM_H
+#define PROTON_GLOBALSCRATCHALLOCOP_TO_LLVM_H
+
+#include "mlir/Conversion/LLVMCommon/TypeConverter.h"
+
+// TODO(crobeck): This pattern is a temporary solution to convert the op
+// to LLVM IR such that we could still have a path to test the frontend. Need to
+// be removed soon.
+
+namespace mlir::triton {
+class TargetInfoBase;
+namespace proton {
+
+void populateGlobalScratchAllocOpToLLVMPattern(LLVMTypeConverter &typeConverter,
```

Comment:
I wonder if the additional argument has been added or not? If yes, we can enable this pattern now, right? I understand that there's no test but that's fine

Reply:
It should be but since we're modifying the function signature I don't want to enable this until we have a test in place since it could create difficult bugs to hunt down while developing the other passes.